docs/node/statesync.mdx

```diff
@@ -1,32 +1,25 @@
 ---
 sidebar_position: 5
-sidebar_label: "Statesync"
+sidebar_label: "State Sync"
 id: "Statesync"
 title: "Using State Sync"
-description: "Learn how to bootstrap a new node using statesync"
+description: "Learn how to bootstrap a new node using state sync"
 slug: /node/statesync
 ---
 
-Any new node joining a network needs to first catch up with the existing nodes. The default approach is to replay all the blocks from the genesis. Depending on the number of blocks in the chain, this process can take a long time.
+Any new node joining a network needs to first catch up with the existing nodes. The default approach is to replay all the blocks from genesis. Depending on the number of blocks in the chain, this process can take a long time.
 
-Statesync is a feature that allows a new node to bootstrap directly from a snapshot instead of replaying all the blocks from the genesis. Statesync greatly reduces the time needed to sync a new node to a network; however, the node will have a truncated history starting from the height of the snapshot.
+State sync is a feature that allows a new node to bootstrap directly from a state snapshot instead of replaying all the blocks from genesis. State sync greatly reduces the time needed to sync a new node to a network; however, the node will have a truncated transaction history starting from the height of the snapshot.
 
-:::note
-Statesync should not be used on a partially synced node. If you have a partially synced node, it switches to blocksync to sync the remaining blocks.
+:::info
+Snapshot creation requires the `pg_dump` tool on the node, while snapshot restoration requires `psql`.
+These are typically included in a `postgresql-client` package. They must be the same version
```

Comment:
Version should be 16. can we add that too. Else few flags in pg_dump aren't supported on 14.

Reply:
Avoiding this docs specificity with "They must be the same version
as the PostgreSQL daemon (`postgres`) that is required by a particular version of Kwil DB"